services/service.go

```diff
@@ -48,6 +50,7 @@ type Service struct {
 // NewService constructor which registers the needed messages.
 func NewService(c *onet.Context) (onet.Service, error) {
 	newUnLynxInstance := &Service{
+		Timeout:          20 * time.Minute,
```

Comment:
This timeout should be configurable. E.g. using the cli library, that way we will be able to pass this through the env for the docker image.